test/parse-outputs/input/duplicate-edges/omicsintegrator1-raw-pathway.txt

```diff
@@ -0,0 +1,4 @@
+A	pd	C
+C	pp	D
+A	pd	C
+C	pp	D
```

Comment:
This test case makes it seem like the util function may be too simplistic. What if the edges were instead:
```
A	pd	C
C	pp	D
A	pp	C
D	pp	C
```
What would the current behavior be? What would the desired behavior be?

Reply:
With that, the code in util.py doesn't take that my thoughts into consideration. It will remove anything in the form A C without taking into account the directionality.